cmd/kubeadm/app/node/csr.go

```diff
@@ -55,3 +62,71 @@ func PerformTLSBootstrap(connection *ConnectionDetails) (*clientcmdapi.Config, e
 
 	return finalConfig, nil
 }
+
+func RequestNodeCertificate(client certclient.CertificateSigningRequestInterface, privateKeyData []byte, nodeName types.NodeName) (certData []byte, err error) {
```

Comment:
Instead add a TODO to `csr.RequestNodeCertificate` that it should use the versioned client and also a TODO to `RequestNodeCertificate()` in `pkg/kubelet/util/csr` to fix this.